sdk/batch/batch-rest/samples-dev/quick-start.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Azure Batch quickstart sample
+ */
+
+import createClient, {
+  BatchClient,
+  CreateJobParameters,
+  CreatePoolParameters,
+  CreateTaskParameters,
+  isUnexpected,
+  paginate,
+} from "@azure-rest/batch";
+import { AzureNamedKeyCredential } from "@azure/core-auth";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const batchAccountEndpoint = process.env["BATCH_ACCOUNT_ENDPOINT"]!;
```

Comment:
We prefer the following style over `!`
```suggestion
const batchAccountEndpoint = process.env["BATCH_ACCOUNT_ENDPOINT"] || "<batch account endpoint>";
```